templates/contests_view.html.twig

```diff
@@ -55,6 +55,7 @@
             <caption style="caption-side: top">{{ msg('scores-legend', [score_calculation_interval~'']) }}</caption>
             <thead>
                 <tr>
+                    <th>{{ msg('rank') }}</th>
```

Comment:
The `rank` message needs to be added to `i18n/en.json` and `i18n/qqq.json`.

Reply:
Okay I got your point, submitting the updated patch very soon! Thanks for your guidance.